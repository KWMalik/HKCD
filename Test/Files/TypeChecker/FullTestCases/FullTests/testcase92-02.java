class foo { public static void main (String[] bar) {int baz = 1 + 3;System.out.println(baz);}}
class bar extends baz { public boolean test() {return 0;}}
class baz { public int test() {return 0;}}
