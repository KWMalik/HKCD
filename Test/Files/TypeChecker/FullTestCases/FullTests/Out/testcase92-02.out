Type error on line 1: Cannot extend the unknown superclass baz.
Type error on line 1: The method {name: <test>, returnType: <boolean>, parameters: <[]>} in bar attempts to override the method {name: <test>, returnType: <int>, parameters: <[]>} in baz.